testing-common/integration-tests/src/main/java/IbmResourceLevelInstrumentationModule.java

```diff
@@ -16,6 +16,11 @@ public IbmResourceLevelInstrumentationModule() {
     super(IbmResourceLevelInstrumentationModule.class.getName());
   }
 
+  @Override
+  public boolean isIndyModule() {
```

Comment:
would it make sense to copy this for indy tests instead of modifying?

Reply:
If my understanding is correct, we can't simply copy the test because the indy and non-indy instrumentations will be both active at the same time and therfore instrument the same class. We'll need to find another class to instrument to do this.

I think that can be done. However, my take was that we are planning to move all instrumentations to the indy-approach soon, therefore it would be okay to already migrate this test. I thought that because we likely won't change the non-indy instrumentation logic anymore we should be safe against regressions even if this test is already migrated.

Another alternative could be to simply migrate this test at a later stage.

Reply:
You could add another test task like in https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/064ba95daaf56d8dccae8dce5df1a130a2e41527/instrumentation/kafka/kafka-clients/kafka-clients-0.11/javaagent/build.gradle.kts#L37 and use system properties to enable the correct instrumentation.